src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/Microsoft.CodeAnalysis.Remote.Razor.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains the Razor design-time infrastructure.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
     <EnableApiCheck>false</EnableApiCheck>
-    <IsShipping>false</IsShipping>
+    <IsShippingPackage>false</IsShippingPackage>
```

Comment:
So `IsShippingPackage` refers to something that is delivered to end-users (needs to be signed etc.) but not turned into a package correct?

Also, I assume this is arcade magic that lights this up?

Reply:
Yep, it's here:

https://github.com/dotnet/arcade/blob/ca7fab569267ed3bc73360882d652d119aae5653/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L5-L24

Reply:
If this is a nuget package then I think this is correct, but in general `IsShipping` [is sufficient](https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#isshipping-isshippingassembly-isshippingpackage-isshippingvsix-bool) and the more precise language is for cases where we ship some sub-set of output artifacts but not all.

Reply:
From @jmarolf's comments, I gather that this pull request is at worst more correct than the state today. Several projects which ship via an installer have been set to non-shipping, which is not correct and undermined some analysis intended to ensure shipping quality.